cli/src/api/load-wasi-template.ts

```diff
@@ -0,0 +1,65 @@
+export const createWasiBinding = (
+  localName: string,
+  wasiRegisterFunctions: string[],
+) => `/* tslint:disable */
```

Comment:
not related with this PR, but I think it's better to remove `tslint` ignore header?